test/system/src/test/java/io/pravega/test/system/PravegaTest.java

```diff
@@ -56,6 +59,8 @@
     private final static String STREAM_SCOPE = "testScopeSampleY";
     private final static String READER_GROUP = "ExampleReaderGroupY";
     private final static int NUM_EVENTS = 100;
+    private final AtomicInteger ReadCount = new AtomicInteger(0);
```

Comment:
There is no need for an AtomicInteger. The build has failed due to incorrect naming convention.

Reply:
the change was done as per the description in the issue (https://github.com/pravega/pravega/issues/2782)
As it is sequential execution, I agree that simply one "int" should be sufficient.
By naming convention, I guess it should name as READ_COUNT, not ReadCount. Correct me if I am wrong.
planning to add private final static int READ_COUNT = 0;

Reply:
Moved the variables to the local function and followed the naming convention of local variables